backend/src/api/automation/automationCreate.ts

```diff
@@ -19,9 +24,25 @@ import track from '../../segment/track'
  */
 export default async (req, res) => {
   new PermissionChecker(req).validateHas(Permissions.values.automationCreate)
+
+  if (!(await isFeatureEnabled(FeatureFlag.AUTOMATIONS, req.currentTenant.id, req.posthog))) {
```

Comment:
Could this be a middleware? Might be easier to use that way?

Reply:
Changed this into a middleware for api endpoints.
Only currently this endpoint is using it as a middleware tho - The other example is in community help center pro, when customUrl exists, I think we can keep it as inline since when customUrl is not existent, we don't really need to call the feature flag